packages/catalog-generator/scripts/build-catalogs.mjs

```diff
@@ -15,13 +15,34 @@ const require = createRequire(import.meta.url);
 const CATALOGS = {
   // https://repo1.maven.org/maven2/org/apache/camel/camel-catalog/
   // https://maven.repository.redhat.com/ga/org/apache/camel/camel-catalog/
-  Main: ['4.9.0', '4.8.2', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
+  Main: [
+    //
+    '4.9.0',
+    '4.8.2',
+    '4.4.4',
+    '4.8.0.redhat-00017',
+    '4.4.0.redhat-00046',
+  ],
   // https://repo1.maven.org/maven2/org/apache/camel/quarkus/camel-quarkus-catalog/
   // https://maven.repository.redhat.com/ga/org/apache/camel/quarkus/camel-quarkus-catalog/
-  Quarkus: ['3.16.0', '3.8.4', '3.8.0.redhat-00018', '3.15.0.redhat-00007'],
+  Quarkus: [
+    //
+    '3.17.0', // Camel 4.9.0
+    '3.15.1', // Camel 4.8.1
```

Comment:
from this blogpost, the Camel 4.8.1 alignment to Camel Quarkus is 3.16.0 https://camel.apache.org/blog/2024/10/camel-quarkus-release-3.16.0/

Reply:
ah but the LTS of Camel Quarkus is the 3.15.x stream

Reply:
and Camel 3.15.1 has also been aligned to 4.8.1

Reply:
Yes, that's correct, but we need to consider that 3.16.0 is not an `LTS` version, whereas 3.15.1 is

```
     4.8.1       3.15.1    17,21  LTS    November 2024   September 2025 
     4.8.1       3.16.0    17,21          October 2024        
```